src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/OverloadResolutionResult.cs

```diff
@@ -860,17 +865,21 @@ private void ReportDuplicateNamedArgument(MemberResolutionResult<TMember> result
             ImmutableArray<Symbol> symbols,
             Location location,
             NamedTypeSymbol typeContainingConstructor,
-            NamedTypeSymbol delegateTypeBeingInvoked)
+            NamedTypeSymbol delegateTypeBeingInvoked,
+            FunctionPointerMethodSymbol functionPointerMethodBeingInvoked)
```

Comment:
>functionPointerMethodBeingInvoked [](start = 40, length = 33)

Instead of having consumer pass this argument, could we simply do a type check on what we have in ```symbols```?